glean-core/src/upload/directory.rs

```diff
@@ -12,9 +12,11 @@ use std::path::{Path, PathBuf};
 use serde_json::Value as JsonValue;
 use uuid::Uuid;
 
-use super::PingRequest;
 use crate::{DELETION_REQUEST_PINGS_DIRECTORY, PENDING_PINGS_DIRECTORY};
 
+/// A representation of the PingData extracted from a ping file.
+type PingData = (String, String, JsonValue);
```

Comment:
*puts on a @Dexterp37-hat*
Naming something [`Data`](https://pharr.org/matt/blog/2020/02/28/naming-maxims.html) is not saying much.
Maybe `PingPayload`?

This being a tuple it's also not clear what is what.
At the very least the doc string needs to tell us that the first part is the document ID, the second the path and the JsonValue the actual JSON-encoded payload.

Reply:
I documented. I agree with a tuple not being very expressive though, it doesn't look too bad when I actually call `process_file`:
```rust
if let Some((document_id, path, body)) =
    self.directory_manager.process_file(document_id)
{
    self.enqueue_ping(&document_id, &path, body, debug_view_tag);
}
```
Do you think it would be better if I make this into a struct?

Reply:
Nah, I think for now the tuple is fine. The documentation on the type alias should help.
And in the code it's more obvious.